R/ml_kmeans.R

```diff
@@ -31,7 +33,7 @@ ml_kmeans <- function(x,
 
   envir <- new.env(parent = emptyenv())
 
-  envir$id <- random_string("id_")
+  envir$id <- sparklyr:::random_string("id_")
```

Comment:
Indeed, we don't need to namespace qualify calls to `sparklyr`s own functions within `sparklyr`.